web/src/settings_profile_fields.js

```diff
@@ -369,6 +369,7 @@ function set_up_select_field_edit_form($profile_field_form, field) {
                 render_settings_profile_field_choice({
                     text: choice.text,
                     value: choice.value,
+                    edit_disabled: true,
```

Comment:
I chose `edit_disabled` over `disabled` since using just `disabled` would not clarify what is disabled without looking at the template file. `disabled` could technically apply to the delete button too.